src/Util/AutoloaderUtil.php

```diff
@@ -72,29 +78,35 @@ public function getNamespacePrefixForClass(string $className): string
         return '';
     }
 
-    private function getClassLoader(): ClassLoader
+    /**
+     * Returns if the namespace is configured by composer autoloader.
+     *
+     * @param string $namespace
+     *
+     * @return bool
+     */
+    public function isNamespaceConfiguredToAutoload(string $namespace)
```

Comment:
Add `: bool` return type, and remove the `@param` and `@return` - they are duplicated by the code itself.